apps/meteor/ee/server/apps/orchestrator.js

```diff
@@ -243,21 +248,6 @@ settingsRegistry.addGroup('General', function () {
 			alert: 'Apps_Logs_TTL_Alert',
 		});
 
-		this.add('Apps_Framework_enabled', true, {
```

Comment:
please remove the setting and the `Apps_Framework_Development_Mode` as well in the migration. otherwise workspaces will still have it showing them on the Admin page

Reply:
@d-gubert

Reply:
I have this change done, was waiting to send it afterwards. But I'll push it now